paddlenlp/datasets/bq_corpus.py

```diff
@@ -0,0 +1,67 @@
+import collections
+import json
+import os
+
+from paddle.dataset.common import md5file
+from paddle.utils.download import get_path_from_url
+from paddlenlp.utils.env import DATA_HOME
+from . import DatasetBuilder
+
+__all__ = ['BQCorpus']
+
+
+class BQCorpus(DatasetBuilder):
+    """
+    BQCorpus: the largest dataset available for for the banking and finance sector
+
+    by frozenfish123@Wuhan University
+
+    """
+    lazy = False
+    URL = "https://dataset-bj.cdn.bcebos.com/qianyan/bq_corpus.zip"
+    MD5 = "abe6c480b96cb705b4d24bd522848009"
+    META_INFO = collections.namedtuple('META_INFO', ('file', 'md5'))
+    SPLITS = {
+        'train': META_INFO(
+            os.path.join('BQCorpus', 'train.tsv'),
```

Comment:
The filepath seems wrong. Have you check your dataset using load_dataset()

Reply:
Sorry actually I've been trapped in the trouble for a long time and I don't know how to fix it ... I've been dealing with it for the past two days, and I can't solve the problem. Sorry for slowing down your progress : (